jumpscale/packages/admin/actors/health.py

```diff
@@ -34,5 +34,17 @@ def get_memory_usage(self) -> str:
     def get_running_processes(self) -> str:
         return j.data.serializers.json.dumps({"data": j.sals.process.get_processes_info()})
 
+    @actor_method
+    def get_health_checks(self) -> str:
+        checks = {"stellar": True}
+        try:
+            j.tools.http.get("https://horizon-testnet.stellar.org")
+            j.tools.http.get(
+                "https://testnet.threefold.io/threefoldfoundation/transactionfunding_service/fund_transaction"
```

Comment:
why this is hard-coded here? and what if we are using the mainnet?

Reply:
Maybe we can have different check according to every network, or show the current identity's network check only?

Reply:
yes I'm just waiting for Waleed's approval about the urls.